updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -46,10 +46,14 @@ module GroupUpdateCreation
       # rubocop:disable Metrics/AbcSize
       # rubocop:disable Metrics/MethodLength
       # rubocop:disable Metrics/PerceivedComplexity
+      # rubocop:disable Metrics/CyclomaticComplexity
       sig { params(group: Dependabot::DependencyGroup).returns(T.nilable(Dependabot::DependencyChange)) }
       def compile_all_dependency_changes_for(group)
         prepare_workspace
 
+        # Raise an error if the package manager version is unsupported
+        dependency_snapshot.package_manager&.raise_if_unsupported!
```

Comment:
When we have unsupported error for any package manager in the group, we are going to raise error before stating update process.